alibi_detect/cd/mmd.py

```diff
@@ -40,6 +41,11 @@ def __init__(
             Backend used for the MMD implementation.
         p_val
             p-value used for the significance of the permutation test.
+        estimator
+            Estimator used for the MMD^2 computation {'quad', 'linear'}. 'Quad' is the default and
+            uses the quadratic u-statistics on each square kernel matrix. 'Linear' uses the linear
+            time estimator as in Gretton et al. (2014), and the threshold is computed using the Gaussian
```

Comment:
Would be helpful to add a link to the paper + reference to the specific section in the paper.

Reply:
Fixed.